lib/htmllint.js

```diff
@@ -23,6 +23,21 @@ module.exports = function( config, done ) {
     return str;
   }
 
+  // parse and, if needed, normalize error messages from HttpClient to vnu.jar format
+  function parseErrorMessages( errors ) {
+    var parsed = JSON.parse( config.server ? '[' + errors.trim().replace( /\n/g, ',' ) + ']' : errors );
+    var messages = parsed.messages;
+    if ( config.server ) {
+      messages = [].concat.apply([], parsed.map(function( file ) {
```

Comment:
The `[].concat.apply` looks like you're concatenating an empty array with the result of the map call - can you skip that and use `messages = parsed.map(...)` directly?